scripts/functions/Function_Layers.js

```diff
@@ -238,6 +237,9 @@ function CLayerSequenceElement() {
     this.m_bRuntimeDataInitialised = false;
     this.m_layer = null;
     this.m_dirtyflags = 0;
+
+    this.m_trackIDToPS = {};

```

Comment:
I think this stuff (and m_trackIDToLastKeyframe) should be part of CSequenceInstance rather than the layer element. It would then mirror how other resources work that are part of a sequence like audio and object instances.